packages/bot-skeleton/src/services/api/api-base.js

```diff
@@ -46,6 +58,7 @@ class APIBase {
     }
 
     reconnectIfNotConnected = () => {
+        this.stopBotIfSocketDisconnected();
```

Comment:
Why are we stopping the bot in the reconnect phase?

Reply:
@balakrishna-deriv will remove this ✅